src/leader_scheduler.rs

```diff
@@ -709,23 +708,16 @@ pub mod tests {
         let num_old_ids = 20;
         let mut old_ids = HashSet::new();
         for _ in 0..num_old_ids {
-            let new_keypair = Arc::new(Keypair::new());
-            let pk = new_keypair.pubkey();
-            old_ids.insert(pk.clone());
-
-            // Give the account some stake
-            bank.transfer(5, &mint_keypair, pk, genesis_block.last_id())
-                .unwrap();
-
             // Create a vote account
-            let vote_signer = VoteSignerProxy::new_local(&new_keypair);
+            let vote_signer = VoteSignerProxy::new();
```

Comment:
Is there a VoteSignerProxy::new()? I thought you last PR changed it to new_with_signer()

Reply:
Yes, I implemented the KeypairUtil trait, which awkwardly defines a `new()`. I pointed it to LocalVoteSigner. By using it here, we're saying that the client-side keypair is ephemeral and not a staking account.

Reply:
The reason I renamed `new(...)` to `new_with_signer(...)` was to make space fo the `KeypairUtil` version, which takes no arguments.

Reply:
👍 

It looks good from vote signing point of view. Also, I think the staking account is correct (same as voting account). @carllin should also review this part.